CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 4.1.0
```

Comment:
Ugh.  The changelog version order is getting messed up.  I'll see if I can fix this again after this PR merges.  No need to hold up the PR on this though.